src/README.md

```diff
@@ -2,64 +2,13 @@
+[SwiftWasm is an open source project to support the WebAssembly target for Swift.](https://github.com/swiftwasm)
 
-The goal of this project is to fully support the WebAssembly target for Swift and to be merged into 
-[the upstream repository](https://github.com/apple/swift).
+The goal of this project is to fully support the WebAssembly target for [Swift](https://swift.org) and to be merged into [the upstream repository](https://github.com/apple/swift).
 
-## Getting Started (with Nightly Toolchain)
+WebAssembly is described on its [home page](https://webassembly.org/) as:
 
-### Installation
+> WebAssembly (abbreviated Wasm) is a binary instruction format for a stack-based virtual machine. Wasm is designed as a portable compilation target for programming languages, enabling deployment on the web for client and server applications.
 
-You can install latest toolchain of SwiftWasm from [Release Page](https://github.com/swiftwasm/swift/releases)
 
-The toolchains can be installed via [`swiftenv`](https://github.com/kylef/swiftenv) like official nightly toolchain.
-
-Here's an example `swiftenv` invocation for installing the latest stable SwiftWasm toolchain on macOS Catalina:
-```sh
-
-$ swiftenv install \
-  https://github.com/swiftwasm/swift/releases/download/swift-wasm-5.3-SNAPSHOT-2020-09-28-a/swift-wasm-5.3-SNAPSHOT-2020-09-28-a-osx.tar.gz
-$ swift --version
-Swift version 5.3-dev (LLVM ba56ef042e, Swift 5855a96018)
-Target: x86_64-apple-darwin19.3.0
-```
-
-### Hello, World!
-
-```sh
-$ echo 'print("Hello, world!")' > hello.swift
-# Compile Swift code into WebAssembly with WASI
-$ TOOLCHAIN_PATH=$(dirname $(swiftenv which swiftc))/../
-$ swiftc \
-    -target wasm32-unknown-wasi \
-    -sdk $TOOLCHAIN_PATH/share/wasi-sysroot \
-    hello.swift -o hello.wasm
-```
-
-You can the run the produced binary with [wasmer](https://wasmer.io/) (or other WebAssembly runtime):
-
-```sh
-$ wasmer hello.wasm
-```
-
-The produced binary depends on WASI which is an interface of system call for WebAssembly.
-So you need to use WASI supported runtime and when you run the binary on browser, you need WASI polyfill library like [@wasmer/wasi](https://github.com/wasmerio/wasmer-js/tree/master/packages/wasi).
-
-### Swift Package Manager
-
-You can also use SwiftPM for managing packages in the same way as other platforms.
-
-```sh
-$ swiftenv local swift-wasm-5.3-SNAPSHOT-2020-09-28-a
-$ swift package init --type executable
-$ swift build --triple wasm32-unknown-wasi
-$ wasmer ./.build/debug/hello-swiftwasm
-Hello, world!
-```
-
-### JavaScript Bindings
-
-[JavaScriptKit](https://github.com/swiftwasm/JavaScriptKit) is Swift framework to interact with JavaScript through WebAssembly.
-
-You can use any JavaScript API from Swift using this library.
+We use LLVM as compiler backend to produce WebAssembly and depends on [WASI](https://wasi.dev), which is a modular system interface for WebAssembly. WASI is mainly used to compile Swift Standard Library.
```

Comment:
```suggestion
We use LLVM as a compiler backend to produce WebAssembly binaries. Our resulting binary also embeds [WASI](https://wasi.dev), which is a modular system interface for WebAssembly. WASI is mainly used to compile Swift Standard Library.
```

Reply:
> Our resulting binary also embeds WASI

I think a produced binary doesn't embed WASI but uses it. 🤔

Reply:
Good point, I've updated the suggestion.